libs/ui/src/pure/SelectDropdown/index.tsx

```diff
@@ -27,4 +27,8 @@ export const SelectDropdown = styled.select`
       linear-gradient(to right, var(--cow-color-lightBlue-opacity-80), var(--cow-color-lightBlue-opacity-80)) 100% 0 /
         26px 100% no-repeat;
   }
+
+  option {
+    background: var(--cow-color-paper);
```

Comment:
Can you use the `UI` enum here (and the rest of the file actually) instead?

Reply:
Fixed